config/settings.yml

```diff
@@ -1446,6 +1446,49 @@ mobile_lighthouse_letters:
   api_scopes:
     - "letters.read"
 
+mobile_lighthouse_benefits_claims:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
+  access_token_url: 'https://sandbox-api.va.gov/oauth2/claims/system/v1/token'
+  api_url: 'https://sandbox-api.va.gov/services/claims/v1/'
+  api_scopes:
+    - "claim.read"
+    - "claim.write"
+
+mobile_lighthouse_letters_2:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
+  access_token_url: 'https://sandbox-api.va.gov/oauth2/va-letter-generator/system/v1/token/token'
+  api_url: 'https://sandbox-api.va.gov/services/va-letter-generator/v1/'
+  api_scopes:
+    - "letters.read"
+
+mobile_lighthouse_direct_deposit:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
+  access_token_url: 'https://sandbox-api.va.gov/oauth2/direct-deposit-management/system/v1/token/token'
+  api_url: 'https://sandbox-api.va.gov/services/direct-deposit-management/v1/'
+  api_scopes:
+    - "direct.deposit.read"
+    - "direct.deposit.write"
+
+mobile_lighthouse_veteran_verification:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
+  access_token_url: 'https://sandbox-api.va.gov/oauth2/direct-deposit-management/v1/token'
+  api_url: 'https://sandbox-api.va.gov/services/veteran_verification/v2'
```

Comment:
I got the api urls for the others from the openapi.json file. In this case, I was pointed to a doc that included this particular url. I suspect that was a misunderstanding and this should be v1.

Reply:
This requires V2 as that is where Client Credentials Grant (system-system) OAuth support is added.